components/listingsPage/utils.js

```diff
@@ -395,6 +396,10 @@ export const AEMCompanyCard = styled(CompanyCard)`
                 font-weight: 700;
                 border: 1px solid #C82A87;
                 box-shadow: 0px 3px 0px rgba(0, 11, 59, 0.25);
+
+                @media ${mediaDetectIE11} {
+                  height: 50px;
```

Comment:
I wonder if it might be better to just use 50px for all browsers? Seems like a bit of a magic number

Reply:
I found that the height value was not needed for modern browsers, didn't really want to introduce it.